src/frontend/Components/Details/Details.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Link, Redirect } from "react-router-dom";
 import './Details.css';
+import Loading from '../Loading-icon/loading-icon'
```

Comment:
Just a subtle details. Following react guidelines, the component files should be named as `PascalCase` instead of `kebap-case`